protobufs/src/cdh.cpp

```diff
@@ -0,0 +1,77 @@
+#include "cdh.hpp"
+
+using namespace std;
+
+typedef bool (*collect_telemetry_function)(cdh::telemetry::TMFrame &frame);
+
+namespace cdh {
+namespace subsystems {
+uint seq_no = 0;
+
+cdh::telemetry::TMFrame collect_all_telemetry() {
+  // TODO: Move this out of the function.
```

Comment:
Not too sure how I can easily move this, I'll know more when I add more integration.